### DISCUSSION TITLE
Big files upload on Strapi

### DISCUSSION THREAD

**Discussion by GuillermoCAB** [strapi/strapi#7913] [Legacy Topics]
0

I'm new to Strapi and im doing a video upload site (front on React). The problem is that i need to upload big video files. I'm on strapi v3.1.1, the first problem was the limitation on the file upload(was 200MB), but i changed the maxFileSize on the node_modules/formidable/lib/incoming_form.js. Now the file seems to upload until the end, but after that the strapi is returning an error:

error RangeError [ERR_FS_FILE_TOO_LARGE]: File size (3371964008) is greater than possible Buffer: 2147483647 bytes at FSReqCallback.readFileAfterStat [as oncomplete] (fs.js:277:11)

I found on strapi Docs and internet that i should change the maxFileLimit in config/environments/development/request.json, but my config folder does not has such folders/files..

![image](https://user-images.githubusercontent.com/49039468/93270689-046bad80-f788-11ea-826f-4dccff493933.png)

**Comment by derrickmehaffy** [comment]
The file you are referencing is old (beta docs), you will need to customize the parser here: https://strapi.io/documentation/v3.x/concepts/middlewares.html#request-middlewares
